src/main/java/net/mcreator/element/types/Enchantment.java

```diff
@@ -46,6 +46,9 @@
 
 	public Enchantment(ModElement element) {
 		super(element);
+
+		this.canGenerateInLootTables = true;

```

Comment:
Check eg. biome how it is implemented there, this is not default constructor. And test your change to see if they work..

Reply:
Well this is how Biome.java is? 
```
public Biome(ModElement element) {
	super(element);

	// RESTORE CONSTANTS (FOR GSON)
	TREES_VANILLA = 0;
	TREES_CUSTOM = 1;

	// DEFAULT VALUES
	name = "";
	spawnStronghold = true;
}
```

Reply:
Yes, but you also need a default constructor or GSON will not pick it up.

https://github.com/MCreator/MCreator/blob/9c803cfcd1a6a015f20f73eeee705cc18d8d1159/src/main/java/net/mcreator/element/types/Biome.java#L106

Reply:
oh